platforms/extensibility/plugin-use/src/main/java/org/gradle/plugin/management/internal/autoapply/DefaultAutoAppliedPluginHandler.java

```diff
@@ -67,7 +67,14 @@ public PluginRequests getAutoAppliedPlugins(PluginRequests initialRequests, Obje
     }
 
     private static PluginRequests filterAlreadyAppliedOrRequested(PluginRequests autoAppliedPlugins, final PluginRequests initialRequests, final PluginContainer pluginContainer, final ScriptHandler scriptHandler) {
-        return PluginRequests.of(ImmutableList.copyOf(StreamSupport.stream(autoAppliedPlugins.spliterator(), false).filter(autoAppliedPlugin -> !isAlreadyAppliedOrRequested(autoAppliedPlugin, initialRequests, pluginContainer, scriptHandler)).collect(Collectors.toList())));
+        return PluginRequests.of(ImmutableList.copyOf(StreamSupport.stream(autoAppliedPlugins.spliterator(), false)
```

Comment:
:x: The alternative coordinates need to be taken into account in `AlreadyOnClasspathIgnoringPluginResolver` as well.

Reply:
Fixed in https://github.com/gradle/gradle/pull/27365/commits/bb1266e15fc7663403e9db7e55e9f8b7199768f2